src/reducers/apiReducer.js

```diff
@@ -0,0 +1,28 @@
+export const initialState = {
+  data: [],
+  status: "",
+  error: "",
+  loading: true,
+};
+
```

Comment:
That's good that you are maintaining `error, loading ` state also

Reply:
Yups thanks